Gemfile.lock

```diff
@@ -312,7 +315,7 @@ DEPENDENCIES
   web-console
 
 RUBY VERSION
-   ruby 3.3.2p78
+   ruby 3.3.4p94
```

Comment:
Don't change this. and other un-required changes in this file, only dotenv-rails should be there.